Add updated SIG diagram

bgrant0607 opened the pull request:
This is the SIG diagram I sent to Steering recently. It contains a couple changes that haven't been completely implemented yet, and I expect more changes soon, but I thought it would be useful to capture a mostly up-to-date snapshot sooner rather than later, since this has never been checked in.

I'll let someone else figure out the best place(s) to drop a link. I considered list.tmpl, but it would push the actual SIG list off the first page in a reasonably sized browser window. Maybe that's ok.

Another possibility would be governance.md, after the explanation of the different types of groups.

Or even README.md.

cblecker commented:
/lgtm
/approve
/hold

Don't know if you want any other reviews, but it looks good to me. Feel free to remove hold when ready for merge.

bgrant0607 commented:
Thanks. I addressed comments from the steering email thread. Can always send another version when more changes are needed, but I wanted to have something in place.

/hold cancel
